Gemfile.lock

```diff
@@ -801,9 +801,3 @@ DEPENDENCIES
   whenever
   wicked_pdf
   wkhtmltopdf-binary
-
-RUBY VERSION
-   ruby 2.1.5p273
-
-BUNDLED WITH
-   1.16.1
```

Comment:
:warning: we do want these lines. What Bundler are you using? :thinking_face: try getting the last one by running `gem update bundler`. This should add them back.